tests/test_track_output/results_test.dat

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<VTKFile type="PPolyData" version="0.1" byte_order="LittleEndian" compressor="vtkZLibDataCompressor">
+  <PPolyData GhostLevel="0">
+    <PPoints>
+      <PDataArray type="Float32" Name="Points" NumberOfComponents="3"/>
+    </PPoints>
+    <Piece Source="poly_0.vtp"/>
+  </PPolyData>
+</VTKFile>
```

Comment:
This was accidentally committed. For some reason this file isn't always cleaned up when I run the test suite locally, and it isn't included in .gitignore.